pipelines/pipeline_navvis_rig.py

```diff
@@ -0,0 +1,305 @@
+import argparse
+import shutil
+import tempfile
+from pathlib import Path
+from typing import List, Optional
+
+from scantools import (
+    logger,
+    run_meshing,
+    run_navvis_to_capture,
+    run_rendering,
+    to_meshlab_visualization,
+)
+from scantools.capture import Capture
+from scantools.run_qrcode_detection import run_qrcode_detection_session
+from scantools.scanners.navvis.camera_tiles import TileFormat
+
+TILE_CHOICES = sorted([attr.name.split("_")[1] for attr in TileFormat])
+
+description = """
+--------------------------------------------------------------------------------
+
+Pipeline description
+====================
+
+Converts Navvis data to Lamar/Capture format exported as rig, including meshing,
```

Comment:
Isn't it Lamar/Kapture format. don't need to change but I want to remark its confusing

Reply:
It is Capture with 'C'. Please see this Readme: [CAPTURE.md](https://github.com/microsoft/lamar-benchmark/blob/main/CAPTURE.md)

Reply:
Yes, it is confusing.